config/sync/config_ignore.settings.yml

```
@@ -7,5 +7,6 @@ ignored_config_entities:
   - 'govdelivery_bulletins.settings:govdelivery_password'
   - 'govdelivery_bulletins.settings:govdelivery_username'
   - system.menu.devel
+  - 'node_link_report.settings:domains_to_skip'
```

Just to keep things alphabetical for finding stuff if this list grows.
```suggestion
  - 'node_link_report.settings:domains_to_skip'
  - system.menu.devel
```